packages/vscode-extension/package.json

```
@@ -206,6 +206,7 @@
           "type": "string",
           "scope": "window",
           "default": "vscode",
+          "deprecationMessage": "This setting is deprecated and Radon IDE panel will use the vscode theme by default.",
```

```suggestion
          "deprecationMessage": "This setting is deprecated and Radon IDE panel will always use the vscode theme.",
```
"By default" implies you can actually change it.